frontend/public/components/factory/table.tsx

```diff
@@ -212,15 +200,16 @@ const VirtualBody: React.SFC<VirtualBodyProps> = (props) => {
 
   return (
     <VirtualTableBody
+      ref={bindBodyRef}
       autoHeight
       className="pf-c-table pf-m-compact pf-m-border-rows pf-c-virtualized pf-c-window-scroller"
       deferredMeasurementCache={cellMeasurementCache}
       rowHeight={cellMeasurementCache.rowHeight}
       height={height || 0}
       isScrolling={isScrolling}
+      isScrollingOptOut={true}
       onScroll={onChildScroll}
-      overscanRowCount={10}
-      columns={columns}
+      columns={[1]}
```

Comment:
simply another way of passing `columnCount={1}`. I forgot to add this option in PF-React, but I'll add it in the future. Since we are technically only caching by rows (and not each column), we can tell react-virtualized to only call our renderer once per row this way.

Reply:
* added `columnCount` to PF and consuming it here now